README.md

```diff
@@ -101,7 +101,7 @@ Each device must have a unique device ID which adhere to the [ID Format](#id-for
 #### Device Attributes
 
 * `homie` / `device ID` / **`$device-attribute`**:
-A device attribute MUST be one of these:
+The set of device attributes MUST be sent immediately after connection to MQTT broker is established and MUST be one of these:
```

Comment:
Yes, I already noticed that while writing - I wanted to avoid the passive sentence.
However, there are various other sections in the same wording, so I didn't want to change them all.
Furthermore, it is better to move the "trigger" to the beginning of the sentence.

So, better:

"When the MQTT connection to the broker is established or re-established, the device MUST sends its attributes to the broker immediately."

Regarding "immediately": I propose to create a new section for handling of the`$online` topic (or `$state` if it gets renamed), that clarifies the requirements for connection establishment (based on #24 ).